ROMFS/px4fmu_common/init.d-posix/17001_TF-G1

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#
+# @name Plane SITL
```

Comment:
More specific name?

Reply:
Renamed in d6ae722 
Added description d6ae7228f560beee834b3f7504bd45c3b28dc8b3